assets/package.json

```diff
@@ -30,10 +27,7 @@
 		"@material/react-tab-bar": "^0.15.0",
 		"@material/react-text-field": "^0.15.0",
 		"@material/ripple": "^2.3.0",
-		"@material/select": "^2.3.1",
 		"@material/switch": "^2.3.0",
-		"@material/textfield": "^2.3.1",
-		"@material/theme": "^1.1.0",
```

Comment:
Why are these removed and others are not?

Reply:
These are not directly needed, they exist as dependencies for `react-` version packages (for example, `@material/react-select` installs `@material/select` as its dependency) and get installed automatically. So, we can safely remove it from our dependencies list to avoid maintaining it directly.